mlflow/tracing/utils/otlp.py

```diff
@@ -181,3 +183,44 @@ def _decode_otel_proto_anyvalue(pb_any_value: AnyValue) -> Any:
     else:
         # For simple types, just get the attribute directly
         return getattr(pb_any_value, value_type)
+
+
+def decompress_otlp_body(raw_body: bytes, content_encoding: str) -> bytes:
+    """
+    Decompress OTLP request body according to Content-Encoding.
+
+    Supported encodings:
+    - gzip
+    - deflate (RFC-compliant and raw deflate)
+
+    Raises HTTPException if the payload cannot be decompressed.
+    """
+    from fastapi import HTTPException, status
+
+    match content_encoding:
```

Comment:
I removed `identity`. According to https://github.com/open-telemetry/opentelemetry-collector/blob/73f090cd0d59a3faa237168750d44a0036d8fd5c/config/configcompression/compressiontype.go#L23, there is no compression named `identity`.